autometa/taxonomy/ncbi.py

```diff
@@ -299,23 +298,15 @@ def parse_names(self):
 
     # @timeit
     def parse_nodes(self):
-        """Short summary.
-
-        Parameters
-        ----------
-        verbose : boolean
-            prints progress to terminal (the default is False).
+        """
+        Parse the `nodes.dmp` database to be used later by :func:`~ncbi.NCBI.parent`, :func:`~ncbi.NCBI.lineage`
```

Comment:
what other methods use `self.nodes`. If you are going to list other methods that use this data structure, you should enumerate all of them.